gui-tk/wiring.tcl

```diff
@@ -650,7 +650,7 @@ proc contextMenu {x y X Y} {
                .wiring.context add command -label "Initialise Random" \
                     -command "initRandom" 
             }
-            .wiring.context add command -label "Copy" -command "canvas.copyItem"
+            .wiring.context add command -label "Copy" -command "minsky.copy"
```

Comment:
This changes the semantics. In the context menu, the copy command means "copy this item". You've changed it to "copy the selection", which is rather confusing, I would think.